not offering previous city names.R

ynwaitl opened the pull request:
i didn't do a lot of new changes from the last modifications. problem should've been solved in previous code, except for adding map again and ensuring country is selected before proceeding. #2 should not be a problem now, when i click to choose new country, city names disapear automatically.

ynwaitl committed:
not offering previous city names.R

i didn't do new changes from the last modifications. problem should've been solved in previous code, except for adding map again.

strepon commented:
There should be only code relevant for the solved problem in this branch - the part with map should not be there.

Now the city does not persist, but when I change the country, I get a pop-up window with the message "No information available for the selected city." - which is a bit disturbing, I would prefer to not show it.

ynwaitl committed:
Update not offering previous city names.R

I modified the app to remove the pop-up, returning an empty search line instead.

ynwaitl committed:
Delete not offering previous city names.R
